expat/CMakeLists.txt

```diff
@@ -611,6 +611,60 @@ else()
 endif()
 string(TOUPPER "${CMAKE_BUILD_TYPE}" _EXPAT_BUILD_TYPE_UPPER)
 
+#
+# packaging   
+#
+include(InstallRequiredSystemLibraries)
+
+set(CPACK_GENERATOR TGZ ZIP)
```

Comment:
I think we don't need both of these at the same time, rather `zip` on Windows only and `TGZ` on non-Windows only.

Reply:
Agreed. Decision was taken over from source project and can be changed to expats needs.